.rubocop.yml

```diff
@@ -18,7 +18,7 @@ Layout/SpaceInsideArrayLiteralBrackets:
 Style/DoubleNegation:
   Enabled: false
 Layout/FirstArrayElementIndentation:
-  EnforcedStyle: 'consistent'
+  EnforcedStyle: "consistent"
```

Comment:
Why did we change this?

Reply:
Because my prettier formatter extension from vscode did this, I think it's not affecting the program while all things are working fine. Anyway, I disable my formatter for future implementations.